pymc_experimental/utils/pivoted_cholesky.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import torch
+
+from gpytorch.utils.permutation import apply_permutation
```

Comment:
I think It'd be best to make `torch` and `gpytorch` optional imports.  

```python
try:
    import torch
    import gpytorch
except ImportError as e:
    raise ImportError("message")
```
That way they don't need to be dependencies for someone who wan'ts to use `pymc_experimental` for something else.

Reply:
Agreed

Reply:
I've looked into the apply_permutation function, I think it is better to avoid the dependency and implement the same function here

Reply:
@ferrine Yes, I plan to use the gpytorch.apply_permutation temporarily and implement the apply_permutation function once I have GP inference working on the GPU.

I have now added the try...catch checks. Is it ok if I check-in this ? I have created a n issue for the same https://github.com/pymc-devs/pymc-experimental/issues/85

Reply:
100% agree @ferrine, longer term that'll have to implemented here, but this PRs more at the proof-of-concept level so lot's of time to go back over those details later.  With the `try...except` it shouldn't get in anyones way.